backend/test/Test/DB/DBSpec.hs

```diff
@@ -0,0 +1,107 @@
+module Test.DB.DBSpec
+  ( spec
+  ) where
+
+import Universum
+
+import qualified Database.Beam.Postgres.Full as Pg
+
+import Database.Beam (all_, insertValues, select)
+import Database.Beam.Backend (SqlSerial(..))
+import Database.Beam.Schema.Tables (primaryKey)
+import Test.Hspec (Spec, describe, it, shouldBe)
+import Test.Hspec.Hedgehog (forAll, modifyMaxSuccess)
+
+import Edna.DB.Integration (runInsert', runSelectReturningOne')
+import Edna.DB.Schema
+  (AnalysisMethodT(..), CompoundT(..), EdnaSchema(..), ExperimentFileT(..), ExperimentT(..),
+  MeasurementT(..), ProjectT(..), SubExperimentT(..), TargetT(..), TestMethodologyT(..),
+  ednaSchema)
+import Test.DB.Gen
+  (genAnalysisMethodRec, genCompoundRec, genExperimentFileRec, genExperimentRec, genMeasurementRec,
+  genProjectRec, genRemovedMeasurementsRec, genSubExperimentRec, genTargetRec,
+  genTestMethodologyRec)
+import Test.Setup (ednaTestMode, withContext)
+
+spec :: Spec
+spec = withContext $ describe "Validate schema" $ modifyMaxSuccess (const 20) $ do
```

Comment:
I think `min 20` is slightly better than `const 20`, because in this case one can at least decrease the number of test runs. So it is 20 by default, but if I want 5 for some reason, I can specify 5 and it will work.